apps/admin-gui/src/app/shared/components/dialogs/edit-application-form-item-dialog/edit-application-form-item-dialog.component.html

```diff
@@ -175,7 +175,11 @@ <h1 mat-dialog-title>
                 <textarea
                   [cdkTextareaAutosize]="true"
                   [(ngModel)]="applicationFormItem.i18n[lang].label"
+                  (input)="checkHtmlContent()"
                   matInput></textarea>
+                <span *ngIf="htmlIsInvalid" class="error">
```

Comment:
Use the 'mat-error' component for the error (see password-form component for inspo)